osmnames/init_database/init_database.py

```diff
@@ -0,0 +1,34 @@
+import os
+from osmnames.helpers.database import exec_sql, exec_sql_from_file, exists
+from osmnames import settings
+
+
+def run():
+    exists_query = "SELECT 1 AS result FROM pg_database WHERE datname='{}'".format(settings.get("DB_NAME"))
+    if exists(exists_query, user="postgres", database="postgres"):
```

Comment:
If you have settings defined somewhere (which it seems you have), why do you still need to pass user and database?